xml/ha_concepts.xml

```diff
@@ -160,6 +160,36 @@
     virtual servers and services running in physical servers. Guest
     systems can be managed as services by the cluster.
    </para>
+   <important>
+    <title>Live migration in &ha; clusters</title>
+    <para>
+     Use caution when performing live migration in an active cluster. The cluster stack
+     might not tolerate an operating system freeze caused by the live migration process,
+     which could lead to the node being fenced.
+    </para>
+    <para>
+     We recommend either of the following actions to help avoid node fencing during live migration:
+    </para>
+    <itemizedlist>
+     <listitem>
+      <para>
+       Increase the &corosync; token timeout and the SBD watchdog timeout, along with
+       any other related settings. The appropriate values depend on your specific setup.
+       For more information, see <xref linkend="sec-ha-storage-protect-watchdog-timings"/>.
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       Before performing live migration, stop the cluster services on either the node or
+       the whole cluster. For more information, see <xref linkend="sec-ha-maint-overview"/>.
```

Comment:
Took a look at the reference again. It seems that this could be more straight-forward for the context: https://documentation.suse.com/sle-ha/15-SP4/single-html/SLE-HA-administration/#sec-ha-maint-outline 

, where it also refers to the section about “Different options for maintenance tasks”.

Otherwise, it looks perfect to me.

Reply:
Sorry, "sec-ha-maint-overview" maybe even more straight-forward, since it refers to the even more details of the specific 
 tasks...

Reply:
No worries, thanks for being so thorough anyway :grin: